internal/config/config_test.go

```diff
@@ -376,6 +376,15 @@ func TestValidate(t *testing.T) {
 				return cnf
 			}(),
 		},
+		"miniup default config is not valid because image and measurements are missing in OSS": {
+			cnf: func() *Config {
+				config, err := MiniDefault()
+				require.Error(t, err)
+				return config
```

Comment:
nit: checking for an error here is redundant since this test is designed to check for validation errors anyway.
What would make more sense is to check if the config returned by `MiniDefault` is not `nil`